charts/backstage/README.md

```diff
@@ -117,9 +117,10 @@ Kubernetes: `>= 1.19.0-0`
 | backstage.annotations | Additional custom annotations for the `Deployment` resource | object | `{}` |
 | backstage.appConfig | Generates ConfigMap and configures it in the Backstage pods | object | `{}` |
 | backstage.args | Backstage container command arguments | list | `[]` |
-| backstage.command | Backstage container command | list | `["node","packages/backend"]` |
+| backstage.command | Backstage container command | list | `["node","packages/backend","--config","app-config.yaml"]` |
 | backstage.containerPorts | Container ports on the Deployment | object | `{"backend":7007}` |
 | backstage.containerSecurityContext | Security settings for a Container. <br /> Ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container | object | `{}` |
+| backstage.environment | Backstage container environment: development, production | string | `"development"` |
```

Comment:
this adds some complexity as you should be able to achieve the same thing by passing  the extra flags you want as arguments:

```yaml
backstage:
  args: ["--config", "app-config.yaml", "--config", "app-config.whatever.yaml"]
```